.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# Syntax and details are here: https://help.github.com/articles/about-codeowners/
+#
+# Global code ownership, fallback
+*                   @mythi @poussa @softwarerecipes
+
+# CI/CD
+.github/workflows/ @mythi @poussa @softwarerecipes
+
+# charts
+/charts/device-plugin-operator/  @mythi @poussa
+/charts/evam-0.7.2/  @softwarerecipes
```

Comment:
remove this line, covered by global.

Reply:
hey, what exactly should be removed?

Reply:
the charts that are not related to device plugins (evam, load-generator, multimodal...)